src/main/resources/assets/techreborn/lang/fr_FR.lang

```diff
@@ -20,23 +20,18 @@ tile.techreborn.implosioncompressor.name=Compresseur à implosion
 tile.techreborn.grinder.name=Broyeur industriel
 tile.techreborn.chunkloader.name=Chargeur de chunk industriel
 tile.techreborn.magicenergyconverter.name=Convertiseur d'énergie magique
-tile.techreborn.assemblingmachine.name=Machine d'assemblage
 tile.techreborn.dieselgenerator.name=Générateur diesel
 tile.techreborn.industrialelectrolyzer.name=Électrolyseur industriel
 tile.techreborn.magicenergyabsorber.name=Absorbeur d'énergie magique
+tile.techreborn.assemblingmachine.name=Machine d'assemblage
```

Comment:
" assembleur " would be better here imo, this also is correct but doesn't really sound good..